ui/e2e/src/pages/Panel.ts

```diff
@@ -48,17 +53,18 @@ export class Panel {
     this.resizeHandle = this.container.locator('..').locator('.react-resizable-handle');
 
     this.figure = this.container.getByRole('figure');
-    this.loader = this.container.getByLabel('Loading');
+    this.loader = this.container.locator('[aria-label*=Loading]');
```

Comment:
This wasn't working quite how I expected -- I think it's focused on input labels. Moving to a more explicit locator.